index.d.ts

```diff
@@ -0,0 +1,77 @@
+declare module "migrate" {
```

Comment:
Since this is a `index.d.ts` file corresponding to a `index.js` file, you don't need the `declare module "migrate"`. In fact, it will interfere if anyone is using a fork, etc.

Reply:
Good catch! The declaration was still there from having initially typed this module in another module. Removed it.